fsharp-backend/src/LibExecution/LibExecution.fsproj

```diff
@@ -16,7 +16,8 @@
     <Compile Include="VendoredTablecloth.fs" />
     <Compile Include="RuntimeTypes.fs" />
     <Compile Include="RuntimeTypesAst.fs" />
-    <Compile Include="DvalReprInternal.fs" />
+    <Compile Include="DvalReprInternalDeprecated.fs" />
+    <Compile Include="DvalReprInternalNew.fs" />
```

Comment:
(nitpick) maybe without "New"? I'm not sure what value the suffix brings. Assumedly we're going to rip out the Deprecated one before long, at which point we'd strip the "new" anyway? Not a big deal either way.